src/TimeIOGroup.cpp

```diff
@@ -216,7 +216,18 @@ namespace geopm
                             "not valid for TimeIOGroup",
                             GEOPM_ERROR_INVALID, __FILE__, __LINE__);
         }
-        return "Time in seconds since the IOGroup load.";
+
+        std::string result = "Invalid signal description: no description found.";
+        // std::string result;
+        result = "    description: Time since the IOGroup load.\n";
```

Comment:
Time since first request of time was handled...

Time since start of application profiling.